packages/fxa-content-server/app/scripts/views/get_flow.js

```diff
@@ -58,9 +63,14 @@ class GetFlowView extends BaseView {
     }
 
     const queryString = Url.objToSearchString({
-      device_id: deviceId,
-      flow_begin_time: flowBeginTime,
-      flow_id: flowId,
+      broker,
+      context,
+      deviceId,
+      flowBeginTime,
+      flowId,
+      isSampledUser,
+      service,
+      uniqueUserId,
```

Comment:
Some things that the metrics endpoint requires are just not set up in fxa-settings, so for now we can steal it during the get_flow process.